src/tools/tidy/src/style.rs

```diff
@@ -136,15 +137,28 @@ macro_rules! suppressible_tidy_err {
     };
 }
 
+pub fn is_in(full_path: &Path, folder_to_find: &str) -> bool {
+    if let Some(parent) = full_path.parent() {
+        if parent.file_name().map(|f| f.to_string_lossy() == folder_to_find).unwrap_or(false) {
```

Comment:
```suggestion
        if parent.file_name().map_or_else(false, |f| f.to_string_lossy() == folder_to_find) {
```
Do we need `to_string_lossy`?

Reply:
We need a conversion to `string` so I took the code from below and didn't change it much.